Troubleshooting guide update (#11896)
openvinotoolkit/openvino: docs/install_guides/troubleshooting-steps.md

## Check the versions of OpenVINO Runtime and Developement Tools

* To check the version of OpenVINO Development Tools, use the following command: <!--updated-->
   ```sh
   mo --version
   ```

> * To check the version of OpenVINO Runtime, use the following command: <!--updated-->

Reviewer: ```suggestion
* To check the version of OpenVINO Runtime, use the following code: <!--updated-->
```

   ```sh
   mo --version
   ```
* To check the version of OpenVINO Runtime, use the following code:
   ```sh
   from openvino.runtime import get_version get_version()
   ```

## Check the versions of Python and PIP

> To check your Python version, run `python -VV` or `python --version`. The supported Python versions should be 64-bit and between 3.7 and 3.9. Note that Python 3.6 is not supported anymore. <!--check with Ryan-->

Reviewer 2: ```suggestion
To check your Python version, run `python -VV` or `python --version`. The supported Python versions should be 64-bit and between 3.7 and 3.9. Starting in 2022.2, Python 3.6 is no longer recommended. Please upgrade.
```

If your Python version does not meet the requirements, update Python:

* For Windows, **do not install Python from a Windows Store** as it can cause issues. You are highly recommended to install Python from <https://www.python.org/>.

> * For Linux and macOS systems, download and install a proper Python version from <https://www.python.org/>. See the [Python Beginners' Guide](https://wiki.python.org/moin/BeginnersGuide/Download) for more information on  version selection. Note that macOS 10.x comes with python 2.7 installed, which is not supported, so you still need install Python from its official website.

Reviewer 2: ```suggestion
* For Linux and macOS systems, download and install a proper Python version from <https://www.python.org/>. See the [Python Beginners' Guide](https://wiki.python.org/moin/BeginnersGuide/Download) for more information selecting a version. Note that macOS 10.x comes with python 2.7 installed, which is not supported, so you must install Python from the official website.
```

## Check if environment variables are set correctly 

> - For Python developers, if you installed OpenVINO using the installer previously, and now are installing OpenVINO with PIP, remove all the PATH settings and the lines with `setupvars` from `.bashrc`. Note that if you installed OpenVINO with PIP in a virtual environment, you do not need to set environment variables.

Reviewer 2: ```suggestion
- For Python developers, if you previously installed OpenVINO using the installer, and are now installing OpenVINO using PIP, please remove all the PATH settings and the lines with `setupvars` from `.bashrc`. Note that if you installed OpenVINO with PIP in a virtual environment, you don't need to set any environment variables.
```

- If you have installed OpenVINO before, you probably have added `setupvars` to your `PATH /.bashrc` or Windows environment variables. After restarting your environment, you should see similar information as below: 
```sh
[setupvars.sh] OpenVINO™ environment initialized
```

>    - If you don't see the information, your PATH variables may be configured incorrectly. Check if you have written the correct <INSTALL_DIR> or tried to activate it in the right folder.

Reviewer 2: ```suggestion
   - If you don't see the information above, your PATH variables may be configured incorrectly. Check if you have typed the correct <INSTALL_DIR> or check if you are trying to activate in the correct directory.
```

   - If you added it to a `.bashrc` file, make sure that the command is correctly written and the file is found in the `~/.bashrc` folder.

> ## Verify if OpenVINO is correctly installed

Reviewer 2: ```suggestion
## Verify that OpenVINO is correctly installed
```

* For Python developers, to verify if OpenVINO is correctly installed, use the following command:
   ```sh
   python -c "from openvino.runtime import Core"
   ```
   If OpenVINO was successfully installed, nothing will happen. If not, an error will be displayed.

> * If you installed OpenVINO Runtime by using the installer, you can search "openvino" in Apps & Features on a Windows system, or check your installation directory on Linux to see if OpenVINO is there.

Reviewer 2: ```suggestion
* If you installed OpenVINO Runtime using the installer, you can search "openvino" in Apps & Features on a Windows system, or check your installation directory on Linux to see if OpenVINO is there.
```

* If you installed OpenVINO Runtime from APT, use the `apt list --installed | grep openvino` command to list the installed OpenVINO packages.

* If you installed OpenVINO Runtime from YUM, use the **yum list installed 'openvino\*'** command to list the installed OpenVINO packages.

## Check if GPU drvier is installed

> [Additional configurations](configurations-header.md) are required in order to use OpenVINO on different hardware.

Reviewer 2: ```suggestion
[Additional configurations](configurations-header.md) may be required in order to use OpenVINO with different hardware like Intel GPUs.
```

> To run inference on GPU, make sure that you have installed the correct GPU driver. To check that, see [additional configurations for GPU](configurations-for-intel-gpu.md).

Reviewer 2: ```suggestion
To run inference on an Intel GPU, make sure that you have installed the correct GPU driver. To check that, see [additional configurations for GPU](configurations-for-intel-gpu.md).
```

> ## Check firewall/network settings

Reviewer 2: ```suggestion
## Check firewall and network settings
```

## Check if environment variables are set correctly 
<!--updated-->
- If you have installed OpenVINO before, you probably have added `setupvars` to your `PATH /.bashrc` or Windows environment variables. After restarting your environment, you should see similar information as below: 
```sh
[setupvars.sh] OpenVINO™ environment initialized
```
   - If you don't see the information, your PATH variable may be configured incorrectly. Check if you have written the correct <INSTALL_DIR> or tried to activate it in the right folder.
   - If you added it to a `.bashrc` file, make sure that the command is correctly written and the file is found in the `~/.bashrc` folder.

> - For Python developers, if you installed OpenVINO using the installer previously, and now are installing OpenVINO with PIP, remove all the PATH settings and the lines with `setupvars` from `.bashrc`. <!--updated-->

Reviewer: Maybe move this up and clarify that if you install OpenVINO with pip in a virtual environment, you do not need to set environment variables.

Author: Done.
